README.md

```diff
@@ -1,34 +1,251 @@
+    no future Features
+- [🤝 Contributing](#contributing)
+    Microverse
+    @bellom
+- [⭐️ Show your support](#support)
+    Thanks to @bellom
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [hello microverse] <a name="hellomicroverse"></a>
+
+> use of html and css a simple web page
+
+**[helo micriverse]** is a test page
+
+## 🛠 Built With <a name="hellomicroverse"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Use of html and css]**
+- **[linters]**
+- **[github]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://github.com/gonzaloalonzo/hellomicroverse)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> You may fork or download the proyect
+
+### Prerequisites
+
+In order to run this project you need:
+
+Browser for internet
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/gonzaloalonzo/hellomicroverse
+```
+
+
+### Install
+
+Install this project with:
+
+    sh
+  cd my-project
+  gem install https://github.com/gonzaloalonzo/hellomicroverse
+
+
+### Usage
+
+To run the project, execute the following command:
+
+https://github.com/gonzaloalonzo/hellomicroverse
+
+### Run tests
+
+To run tests, run the following command:
+
+    sh
+  bin/rails test test/models/article_test.rb
+
+
+### Deployment
+
+You can deploy this project using:
+
+https://github.com/gonzaloalonzo/hellomicroverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+Gonzalo alonzo, Julio Cezar
+
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/gonzaloalonzo/hellomicroverse)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
```

Comment:
- [OPTIONAL] I would advise you to delete this second author part because you are the only author for now. When you have a new collaborator, you can add the details of the collaborator there. 👍 😎